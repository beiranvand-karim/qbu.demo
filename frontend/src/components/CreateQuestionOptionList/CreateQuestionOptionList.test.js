/* global describe, it, expect */
import React from "react"
import {CreateQuestionOptionList} from "./"
import {shallow} from "enzyme"
import renderer from 'react-test-renderer'
import {MemoryRouter} from "react-router-dom"

describe('<CreateQuestionOptionList />', () => {
   it('should render without crashing', () => {
      shallow(<CreateQuestionOptionList />)
   });

   it('should matches the snapshot', function () {
      const tree = renderer.create(<MemoryRouter><CreateQuestionOptionList /></MemoryRouter>).toJSON();
      expect(tree).toMatchSnapshot()
   });
});