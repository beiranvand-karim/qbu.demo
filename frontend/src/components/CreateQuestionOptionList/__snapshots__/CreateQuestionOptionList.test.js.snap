// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<CreateQuestionOptionList /> should matches the snapshot 1`] = `null`;
